README.md

```diff
@@ -84,6 +87,7 @@ Yeoman, Gulp, Bower, AngularJS, Ionic & of course Cordova. All in one sexy gener
 - [Developing on Windows](https://github.com/mwaylabs/generator-m-ionic/tree/master/docs/guides/windows.md), what you need to know.
 - [Git integration](https://github.com/mwaylabs/generator-m-ionic/tree/master/docs/guides/git_integration.md), see how it's done.
 - [SASS integration](https://github.com/mwaylabs/generator-m-ionic/tree/master/docs/guides/sass_integration.md) in our module concept.
+- [JADE integration](https://github.com/mwaylabs/generator-m-ionic/tree/master/docs/guides/jade_integration.md) in our module concept.
```

Comment:
I don't see this as a guide under **App Development**, because during development it's rather transparent whether you use HTML or Jade, as far as gulp tasks is concerned. Maybe something of the likes of *Choosing HTML or Jade Templating* under **Generation**?